### PR TITLE
Cancel popup

### PR DESCRIPTION
**Pull request by sowmika148**

### CODE REVIEW

File: ui/react-components/components/CustomPopup/CustomPopup.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import PropTypes from "prop-types";
+import classNames from 'classnames';
+import {customPopup} from './CustomPopup.module.scss';
+
+const CustomPopup = props => {
+
+    const {triggerComponent, popupContent} = props;
+
+    const contentStyle = {
+        width: "470px",
```
Comment (binduak) [inline ui/react-components/components/CustomPopup/CustomPopup.jsx]: @sowmika148 Any specific reason for not moving this style to css file

Comment (venkatavineela) [reply ui/react-components/components/CustomPopup/CustomPopup.jsx]: Cannot be moved. We can discuss

File: ui/react-components/components/CancelConfirmation/CancelConfirmation.jsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+                    <i className={classNames("fa", "fa-times")} onClick={close}/>
+                </a>
+            </div>
+            <div>
+                <h1 className={classNames(cancelModalTitle)}>
+                    <FormattedMessage id={'APPOINTMENT_CANCEL_CONFIRMATION_TITLE'} defaultMessage={'Wait!'}/>
+                </h1>
+                <div className={classNames(cancelModalBody)}>
+                    <FormattedMessage id={'APPOINTMENT_CANCEL_CONFIRMATION_TEXT'}
+                                      defaultMessage={'Are you sure you want to cancel adding the new appointment?This will erase everything you have filled. Nothing will be saved.'}/>
```
[the remark this answers is not in the document]
Comment (venkatavineela) [reply ui/react-components/components/CancelConfirmation/CancelConfirmation.jsx]: Where do you want us to include them?

File: ui/react-components/components/CustomPopup/CustomPopup.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (venkatavineela) [reply ui/react-components/components/CustomPopup/CustomPopup.jsx]: Will check if can be moved

File: ui/react-components/components/CustomPopup/CustomPopup.module.scss [hunk 3]
```diff
@@ -0,0 +1,6 @@
+.customPopup {
+
+  div[class^='popup-content'] {
+    margin-top: 7% !important;
```
[the remark this answers is not in the document]
Comment (venkatavineela) [reply ui/react-components/components/CustomPopup/CustomPopup.module.scss]: Popup component has inline styles. To override them we have given important

File: ui/react-components/components/CustomPopup/CustomPopup.jsx (hunk 1 above)
Comment (sowmika148) [reply ui/react-components/components/CustomPopup/CustomPopup.jsx]: Done

File: ui/react-components/components/CancelConfirmation/CancelConfirmation.module.scss [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+      line-height: normal;
+      color: $secondary-color-2;
+    }
+    .cancelModalBody {
+      margin: 0 0 24px;
+      line-height: normal;
+    }
+
+    .button {
+      background: #f2f2f2;
```
[the remark this answers is not in the document]
Comment (sowmika148) [reply ui/react-components/components/CancelConfirmation/CancelConfirmation.module.scss]: Will do along with Success confirmation

File: ui/reactAngularAdaptor.js [hunk 5]
```diff
@@ -3,3 +3,15 @@ import AddAppointmentContainer from './react-components/containers/AddAppointmen
...
 angular.module('bahmni.appointments')
 .component('reactAddAppointment', react2angular(AddAppointmentContainer));
+
+angular.module('bahmni.appointments').component('reactAddAppointmentWrapper',{
+    template: '<react-add-appointment on-back="onBack">',
+    controller: reactAddAppointmentController
+});
+
+reactAddAppointmentController.$inject = ['$scope', '$state'];
+function reactAddAppointmentController($scope, $state) {
```
[the remark this answers is not in the document]
Comment (sowmika148) [reply ui/reactAngularAdaptor.js]: Added comment

**Pull request merged**